libc/optee/syscalls.c

```diff
@@ -3,20 +3,273 @@
 
```

Comment:
This file **optee/syscalls.c** is a cut-n-paste of **sgx/syscalls.c**. We should probably refactor **sgx/syscalls.c** into a common and non-common parts.

Reply:
Indeed; `syscalls.c` is now shared.

Given the recent changes in master, there no longer is an issue with the fact that `SYS_open` does not exist in AArch64 and is instead replaced with `SYS_openat` (a difference that follows Linux).